2/debian-10/rootfs/opt/bitnami/scripts/libkafka.sh

```diff
@@ -91,12 +92,12 @@ kafka_producer_consumer_conf_set() {
 kafka_env() {
     cat <<"EOF"
 export KAFKA_BASEDIR="/opt/bitnami/kafka"
+export KAFKA_VOLUMEDIR="/bitnami/kafka"
 export KAFKA_HOME="$KAFKA_BASEDIR"
 export KAFKA_LOGDIR="$KAFKA_BASEDIR"/logs
-export KAFKA_CONFDIR="$KAFKA_BASEDIR"/conf
-export KAFKA_CONF_FILE_NAME="server.properties"
-export KAFKA_CONF_FILE="$KAFKA_CONFDIR"/$KAFKA_CONF_FILE_NAME
-export KAFKA_VOLUMEDIR="/bitnami/kafka"
+export KAFKA_CONFDIR="$KAFKA_BASEDIR"/config
+export KAFKA_CONF_FILE="$KAFKA_CONFDIR"/server.properties
+export KAFKA_MOUNTED_CONFDIR="${KAFKA_MOUNTED_CONFDIR:-${KAFKA_VOLUMEDIR}/config}"
```

Comment:
We don't usually make the `MOUNTED_CONFDIR` customizable for other containers. I think it is fine leaving it constant:
```suggestion
export KAFKA_MOUNTED_CONFDIR="${KAFKA_VOLUMEDIR}/config"
```

Do you have any use cases for this?

Reply:
I don't have any use cases for this, but it was in the Casandra example I was working from.